frameworks/elastic/src/main/dist/elasticsearch.yml

```diff
@@ -19,6 +19,18 @@ bootstrap.memory_lock: false
 metrics.statsd.host: {{STATSD_UDP_HOST}}
 metrics.statsd.port: {{STATSD_UDP_PORT}}
 
+{{#SHARD_ALLOCATION_AWARENESS_ENABLED}}
+node.attr.region: ${REGION}
```

Comment:
I know "region" here is just a label and could also be "foo", but I hope nobody is actually planning on sharding their ES cluster across geographic regions, like in the AWS sense of the word.

Reply:
Yup. I also noticed we don't have any [region placement constraints](https://github.com/mesosphere/marathon/pull/5574) for the ES nodes by default, does it make sense that we do?

Reply:
Is this mustache templated? Should this not be `{{REGION}}` and `{{ZONE}}`?

Reply:
No, this is an ES feature for [interpolating environment variables into the config file](https://www.elastic.co/guide/en/elasticsearch/reference/current/settings.html#_environment_variable_substitution), an alternative to passing settings through the command line invocation.

Reply:
OK. Does this mean that we're mixing two forms of injection here? Is this desired?

Reply:
It's less than ideal. I was able to use the marathon templating for this instead, just had to fix the tests: https://github.com/mesosphere/dcos-commons/pull/1855/commits/76b5a1b9990ce60e035ad8a585b3c02e63fe3c7d